flocker/control/schema/types.yml

```diff
@@ -59,7 +59,17 @@ definitions:
         external:
           type: integer
     uniqueItems: true
-    
+
+  environment:
+    title: "Container environment variables"
+    description: "Mapping of label and value pairs for environment variables"
+    type: object
+    patternProperties:
+      "^.{1,256}$":
```

Comment:
Is this for the environment variable key or for the value or both?

It seems a little too permissive. 

I checked HybridCluster schema for similar types, but couldn't find anything. But double check this.